render/gles2/renderer.c

```diff
@@ -205,7 +210,20 @@ static void gles2_render_ellipse_with_matrix(struct wlr_renderer *wlr_renderer,
 
 	glUniformMatrix3fv(renderer->shaders.ellipse.proj, 1, GL_FALSE, transposition);
 	glUniform4f(renderer->shaders.ellipse.color, color[0], color[1], color[2], color[3]);
-	draw_quad();
+
+
+	glVertexAttribPointer(renderer->shaders.ellipse.pos_loc, 2, GL_FLOAT,
+			GL_FALSE, 0, verts);
+	glVertexAttribPointer(renderer->shaders.ellipse.tex_loc, 2, GL_FLOAT,
+			GL_FALSE, 0, texcoord);
+
+	glEnableVertexAttribArray(0);
+	glEnableVertexAttribArray(1);
```

Comment:
Shouldn't these use the *_loc members too?

Reply:
It could yes, do you want me to change it?

Reply:
Yes.

Reply:
Done for all calls to `glEnableVertexAttribArray`